packages/website/ts/components/ui/text.tsx

```diff
@@ -26,6 +27,7 @@ export const Text = styled(PlainText)`
     ${props => (props.lineHeight ? `line-height: ${props.lineHeight}` : '')};
     ${props => (props.center ? 'text-align: center' : '')};
     color: ${props => props.fontColor};
+    min-height: ${props => props.minHeight};
```

Comment:
can you wrap the entire rule in the closure since you're not providing a default value? (like 'text-align')